Sending_SMS/README.md

```diff
@@ -0,0 +1 @@
+This a simple python script to send SMS to any desired phone number through Sinch messaging service (free of cost).

```

Comment:
@soumya-ranjan7 Can you please add a link to Sinch API, where user can sign up and get a API Key to use your script.